crossbeam/datasets/run_deepcoder_gen.sh

```diff
@@ -36,9 +37,9 @@ valid_file=$out_dir/valid-tasks.pkl
 python3 -m crossbeam.datasets.bottom_up_data_generation \
     --domain=deepcoder \
     --output_file=$valid_file \
-    --data_gen_seed=1 \
+    --data_gen_seed=10000 \
```

Comment:
we need to make sure the range of seeds for validation is disjoint from the range of seeds for training. these ranges will depend on how many CPUs/processes we use. just keep in mind

Reply:
sure, I use different seed ranges when I do distributed data gen